app/Http/Controllers/RegistrationController.php

```diff
@@ -33,16 +34,17 @@ public function register(Request $request){
         $response["data"]["name"]= $appUser->name;
         $response["data"]["email"]= $appUser->email;
         $response["data"]["contact"]= $appUser->contact;
-        $response["data"]["id"]= $appUser->unique_id;
+        $response["data"]["id"]= $appUser->fcmToken;
+        $response["data"]["rollNo"]= $appUser->rollNo;
 
         return response()->json([
             "status"=>"OK",
             "data"=>[
-            'id'=> $appUser->id,
```

Comment:
Need update on app side. You dont need to remove this

Reply:
changed app backend. Won't ping `id` in POST request